projects/cloud/app/graphql/mutations/change_user_role.rb

```diff
@@ -0,0 +1,17 @@
+module Mutations
+  class ChangeUserRole < ::Mutations::BaseMutation
+    argument :user_id, ID, required: true
+    argument :organization_id, String, required: true
+    argument :role, Types::RoleType, required: true
+
+    type Types::UserType
+
+    def resolve(attributes)
+      begin
+        ChangeUserRoleService.call(**attributes, acting_user: context[:current_user])
```

Comment:
Is `acting_user` the user trying to change the role? If so, what about calling it `role_changer`?